Description: Use system-wide support libraries
 only applicable in concert with other means of ensuring availability of
 the libraries (done in packaging meta hints for Debian)
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2009-12-01
--- a/setup.py
+++ b/setup.py
@@ -307,19 +307,20 @@
         'MoinMoin.security',
         'MoinMoin.stats',
         'MoinMoin.support',
-        'MoinMoin.support.flup',
-        'MoinMoin.support.flup.client',
-        'MoinMoin.support.flup.server',
-        'MoinMoin.support.pygments',
-        'MoinMoin.support.pygments.filters',
-        'MoinMoin.support.pygments.formatters',
-        'MoinMoin.support.pygments.lexers',
-        'MoinMoin.support.pygments.styles',
-        'MoinMoin.support.werkzeug',
-        'MoinMoin.support.werkzeug.contrib',
-        'MoinMoin.support.werkzeug.debug',
-        'MoinMoin.support.xappy',
-        'MoinMoin.support.parsedatetime',
+        ## In Debian, below libraries are packaged separately
+        #'MoinMoin.support.flup',
+        #'MoinMoin.support.flup.client',
+        #'MoinMoin.support.flup.server',
+        #'MoinMoin.support.pygments',
+        #'MoinMoin.support.pygments.filters',
+        #'MoinMoin.support.pygments.formatters',
+        #'MoinMoin.support.pygments.lexers',
+        #'MoinMoin.support.pygments.styles',
+        #'MoinMoin.support.werkzeug',
+        #'MoinMoin.support.werkzeug.contrib',
+        #'MoinMoin.support.werkzeug.debug',
+        #'MoinMoin.support.xappy',
+        #'MoinMoin.support.parsedatetime',
         'MoinMoin.theme',
         'MoinMoin.userform',
         'MoinMoin.userprefs',
